bin/configure-monitoring-secret.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+# This script creates SSM parameter for storing integration URL for incident management
+# services. Script creates new SSM attribute or updates existing. 
+#
+# Positional parameters:
+#   APP_NAME (required) – the name of subdirectory of /infra that holds the
+#     application's infrastructure code.
+#   ENVIRONMENT is the name of the application environment (e.g. dev, staging, prod)
+#   URL is the url for te integration endpoint for external incident management
+#     tools (e.g. Pagerduty, Splunk-On-Call
+# -----------------------------------------------------------------------------
+set -euo pipefail
+
+APP_NAME=$1
+ENVIRONMENT=$2
+INTEGRATION_ENDPOINT_URL=$3
+
+SECRET_NAME=$(./bin/get_monitoring_secret_name.sh | sed "s/\$APP_NAME/$APP_NAME/" | sed "s/\$ENVIRONMENT/$ENVIRONMENT/")
```

Comment:
Use `terraform output` to pull out config values from the project-config or app-config modules. Don't parse the files directly.

For example you can use the pattern here as a reference: https://github.com/navapbc/template-infra/blob/main/bin/run-database-migrations.sh#L39

Reply:
thanks for the great example. I was looking for it, but didnt find it. so I did what I did.

I'm pulling name from app-config now.